docs/ecosystem-user-event.mdx

```diff
@@ -166,7 +166,7 @@ test('delete characters within the selectedRange', () => {
   render(
     <div>
       <label htmlFor="my-input">Example:</label>
-      <input id="my-input" type="text" value="This is a bad example" />
+      <input id="my-input" type="text" defaultValue="This is a bad example" />
```

Comment:
The input needs to be un-controlled for the example to work (see [codesandbox](https://codesandbox.io/s/rtl-selection-range-example-c17hn?file=/src/__tests__/index.js)).